package.json

```
@@ -73,10 +73,12 @@
   "scripts": {
     "babel:build": "babel src -d js --source-maps -q",
     "babel:watch": "babel src -d js --watch --source-maps",
-    "build:w3c": "node ./tools/builder.js --profile=w3c",
-    "build:geonovum": "node ./tools/builder.js --profile=geonovum",
     "build:components": "npm run copydeps && npm run babel:build",
+    "build:geonovum-debug": "node ./tools/builder.js --profile=geonovum --debug",
```

How about this to reduce code duplication?

```suggestion
    "build:geonovum-debug": "npm run build:geonovum -- --debug",
```

---

TIL :)